.devcontainer/Dockerfile

```
@@ -98,6 +97,13 @@ RUN echo "Install general purpose packages" && \
 
 RUN DEBIAN_FRONTEND=noninteractive apt-get install -y clangd-12
 
+# Install golang 1.18
+WORKDIR /usr/local
+RUN curl https://artifactory.magmacore.org/artifactory/generic/go1.18.linux-amd64.tar.gz --remote-name --location && \
+    tar -xzf go1.18.linux-amd64.tar.gz && \
+    rm go1.18.linux-amd64.tar.gz && \
+    export PATH=$PATH:/usr/local/go/bin
```

This export does not work that way (it is lost after that line), instead you need to write:

```suggestion
ENV PATH=$PATH:/usr/local/go/bin
```

---

Oh yes, thanks!